### PR TITLE
Telemetry docs

### PR DESCRIPTION
**Pull request by matthewp**
This is a companion to https://github.com/withastro/astro/pull/3256, documenting how to enable/disable anonymous telemetry.

### CODE REVIEW

**Comment by tony-sull** [comment]
I'm not sure if it would make more sense on the docs site or astro.build, but it would be good for us to add a Telemetry page as well that goes into detail on what is collected and how it's used (similar to [Next.js's telemetry page](https://nextjs.org/telemetry))

File: src/pages/en/reference/cli-reference.md [hunk 1]
```diff
@@ -37,6 +37,28 @@ Runs diagnostics (such as type-checking) against your project and reports errors
 
 This command is intended to be used in CI workflows.
 
+### astro telemetry
+
+Sets telemetry configuration for the current user. Telemetry is anonymous data that provides insights into what features are most often used.
+
+Telemetry can be disabled with this cli command:
```
Comment (delucis) [inline src/pages/en/reference/cli-reference.md:44]: Nit:

```suggestion
Telemetry can be disabled with this CLI command:
```

File: src/pages/en/reference/cli-reference.md [hunk 2]
```diff
@@ -37,6 +37,28 @@ Runs diagnostics (such as type-checking) against your project and reports errors
...
+```shell
+astro telemetry enable
+```
+
+The `clear` command resets the telemetry data:
+
+```shell
+astro telemetry clear
+```
+
```
Comment (delucis) [inline src/pages/en/reference/cli-reference.md:61]: As per https://github.com/withastro/astro/pull/3256#issuecomment-1115219301, maybe worth flagging here how to handle CI:

```suggestion

> Want to disable telemetry in CI environments? Make sure you add the `astro telemetry disable` command to your CI scripts.

```

**Review by delucis — COMMENTED** [review]
LGTM! Left a couple of small comments.

File: src/pages/en/reference/cli-reference.md [hunk 3]
```diff
@@ -37,6 +37,30 @@ Runs diagnostics (such as type-checking) against your project and reports errors
 
 This command is intended to be used in CI workflows.
 
+### astro telemetry
+
+Sets telemetry configuration for the current user. Telemetry is anonymous data that provides insights into what features are most often used.
```
Comment (sarah11918) [inline src/pages/en/reference/cli-reference.md:42]:
```suggestion
Sets telemetry configuration for the current user. Telemetry is anonymous data that provides insights into which features are most often used.
```
Nit.

**Comment by sarah11918** [comment]
I would make the tiny change above, but LGTM!

**Pull request merged**